tests/session/session_error-001.phpt

```diff
@@ -0,0 +1,59 @@
+--TEST--
+MongoDB\Driver\Session with wrong defaultTransactionOptions
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+
+$options = [
+    [ 'readConcern' => 42 ], 
+    [ 'readConcern' => new stdClass ],
+    [ 'readConcern' => new \MongoDB\Driver\WriteConcern( 2 ) ],
+    [ 'readPreference' => 42 ], 
+    [ 'readPreference' => new stdClass ],
+    [ 'readPreference' => new \MongoDB\Driver\ReadConcern( \MongoDB\Driver\ReadConcern::LOCAL ) ],
+    [ 'writeConcern' => 42 ], 
+    [ 'writeConcern' => new stdClass ],
+    [ 'writeConcern' => new \MongoDB\Driver\ReadPreference( \MongoDB\Driver\ReadPreference::RP_SECONDARY ) ],
```

Comment:
Likewise, this can include test cases I suggested in https://github.com/mongodb/mongo-php-driver/pull/850/files#r196199595 (redundant, but not harm in keeping the tests consistent).

Reply:
Done